### ISSUE TITLE
Focusing on an application does not enter Forms Mode (JAWS 2023.2307.37)

### ISSUE THREAD

**Issue by pgouldmiro** [FreedomScientific/VFO-standards-support#894]
### Summary
In JAWS 2023.2307.37, focusing on an application (`role="application"`) does not enter Forms Mode. This means that keyboard shortcuts within the application are intercepted by JAWS, and not passed to the browser.

### Findings
I have created an example webpage to demonstrate the bug:
https://codepen.io/petergmiro/pen/PwoxWYM

The webpage has the following:
1. A button, an application, and another button
2. JS to cycle through the elements on `Tab`
3. JS to enter the application on `Cmd/Ctrl + Shift + Down` (in this test, it just appends " - Entered" to the application text), and exit the application on `Cmd/Ctrl + Shift + Up` (in this test, it just resets the application text)

This webpage is a basic representation of the structure of our web application.

#### JAWS 2023.2307.37  ❌
On JAWS 2023.2307.37, JAWS remains in Virtual PC Cursor Mode when the application is focused and the application cannot be interacted with. Keyboard shortcuts within the application are intercepted by JAWS and not passed to the browser.

[Video on JAWS 2023.2307.37](https://drive.google.com/file/d/1RYwX0V5ek2KaJvM2Xg9dNNzW1CIN2Ilg/view)

#### JAWS 2025.2503.39  ✅
On JAWS 2025.2503.39, JAWS enters Forms Mode when the application is focused. Keyboard shortcuts work within the application. 

[Video on JAWS 2025.2503.39](https://drive.google.com/file/d/1eVPvsuecsLW88lnCZ_PgDLhyZQa9Yasr/view)

### Steps to reproduce
1. Open JAWS 2023.2307.37
2. Open https://codepen.io/petergmiro/pen/PwoxWYM
3. Press `Tab` to move through the elements on the page
4. Focus on the application
5. Attempt to enter with the application using `Ctrl + Shift + Down`

### Expected behaviour
When the application is focused, JAWS should enter Forms Mode. Keyboard shortcuts should not be intercepted by JAWS, and passed to the browser. This is what happens in JAWS 2025.2503.39.

### Actual behaviour
When the application is focused, JAWS remains in Virtual PC Cursor Mode. Keyboard shortcuts are intercepted by JAWS, and not passed to the browser.

### Environment
- OS: Windows 11 Pro 24H2 26100.3476 (latest version at time of writing this bug)
- Browser: Google Chrome 134.0.6998.178 (latest version at time of writing this bug)
- Screen reader: JAWS 2023.2307.37

**Comment by JAWS-test** [comment]
I cannot reproduce the problem with Chrome and JAWS 2023. In my test environment, JAWS switches correctly to application mode and the keyboard operation also works as described in the message

**Comment by pgouldmiro** [comment]
Hi @JAWS-test. Thanks for your reply.

I can reproduce this consistently on JAWS 2023.2307.37. What information can I provide to help you with debugging please?

Thanks.

**Comment by JAWS-test** [comment]
Hi @pgouldmiro: I am not from Freedom Scientific, I just wanted to share my test results with you. Someone from Freedom Scientific will have to find the cause of the error

**Comment by JeremiahRogersVispero** [comment]
Cannot reproduce with JAWS 2025.2506.170 and Chrome 138.0.7204.169. If you can reproduce with JAWS 2025, please share revised code or specific instructions to reproduce and reopen the issue.
